sky/exceptions.py

```diff
@@ -215,3 +215,35 @@ class ClusterOwnerIdentityMismatchError(Exception):
 class NoCloudAccessError(Exception):
     """Raised when all clouds are disabled."""
     pass
+
+
+class AWSAzFetchingError(Exception):
```

Comment:
(not feeling strongly) nit: AwsAzFetchingError

https://google.github.io/styleguide/cppguide.html#General_Naming_Rules